src/Hooks/useForm.js

```diff
@@ -0,0 +1,35 @@
+import { useState } from "react";
+import { omit } from "lodash";
+
+const useForm = () => {
```

Comment:
this is amazing, really great thinking of making a helper function for form handling, really good.

Reply:
you can also make it dynamic instead of making it specific for just two values